pipeline/transformer/transformer_tei/src/main/scala/weco/pipeline/transformer/tei/TeiXml.scala

```diff
@@ -1,13 +1,10 @@
 package weco.pipeline.transformer.tei
 
-import weco.catalogue.internal_model.languages.Language
-
 import scala.util.Try
 import scala.xml.{Elem, Node, XML}
-import cats.implicits._
 import grizzled.slf4j.Logging
 
-class TeiXml(xml: Elem) extends Logging {
+class TeiXml(val xml: Elem) extends Logging {
```

Comment:
Why val?

Reply:
So we can access it elsewhere – if I have an instance of `val t = TeiXml`, I can access the elem with `t.xml` – see what I'm doing in TeiDataParser.